mobile/lib/flashcard/show/screens/flashcard_page.dart

```diff
@@ -48,13 +51,39 @@ class _FlashcardPageState extends State<FlashcardPage> {
                     children: <Widget>[
                       const HeaderFlashcard(),
                       Expanded(
-                        flex: 6,
-                        child: FlashcardContainer(flashcard!.content, flashcard.answer),
-                      ),
-                      Expanded(
-                        flex: 1,
-                        child: Container(),
+                        flex: 8,
+                        child: FlashcardContainer(
+                            flashcard.content, flashcard.answer),
                       ),
+                      Padding(
+                          padding: const EdgeInsets.symmetric(
+                              vertical: 8.0, horizontal: 20),
+                          child: FutureBuilder(
+                            future: tokenStorage.read(key: 'token'),
```

Comment:
tutaj ten kod będzie do zamiany całkowicie, ponieważ będzie to fetchowane z api nie z local storage